scripts/create-readmes.mjs

```diff
@@ -94,6 +94,16 @@ function variablePlugin(opts) {
           node.value = node.value.replace("$", n)
         })
       }
+
+      // support variable file name extensions in inline code
+      if (node.type === "inlineCode") {
+        if (!opts.fileExt) {
+          throw new Error(
+            `Inline Code has variable file extension but variable value was not provided`
+          )
+        }
+        node.value = node.value.replace(":var[fileExt]", opts.fileExt)
+      }
```

Comment:
@jxnblk here's the new variable replace code I had to put in. Curious if you think this should be done in another manner

Reply:
This looks close; if you look at line 52, that's where the value is being parsed. When working with ASTs, you generally want to avoid things like string replace and regex because it's already been parsed (the replace on 94 is kind of a hack). I'd need to look a little closer to refresh my memory, but I think we probably want this logic in the block above when it visits the `var` node on line 56. There might be something a little more needed since this is for `inlineCode` though